src/BrazeEndOfArticleComponent.ts

```diff
@@ -1,29 +1,46 @@
 import React from 'react';
 
+// Common Epic imports
+import {
+    buildBrazeMessageComponent,
+    ComponentMapping,
+    HasConsolidatedTrackClick,
+} from './buildBrazeMessageComponent';
+import type { BrazeClickHandler, SubmitComponentEvent } from './utils/tracking';
+
+// Why are we mapping this component?
+// Isn't it the blueprint on top of which other newsletter epics get built?
```

Comment:
This is a good question. Yep you're right it's the blueprint, but this can be the generic version which marketing would like. I think the main reason it's not usable already is related to tooling (for example making selecting an image really easy - see the embedded grid image picker approach used by the `BannerWithLink` component)

Reply:
Good thought for the upcoming work, though I suggest we create a new folder for the new self-serve template so existing canvases/campaigns don't break when we develop/deploy it. Then Marketing can migrate canvases at their leisure.